src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,27 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
```

Comment:
```suggestion
        { user && <UserInfo user={user} />}
```
technically user can be undifined and in such case UserInfo component will fail.